### PR TITLE
[FLINK-27797][API/Python] PythonTableUtils.getCollectionInputFormat c…

### PR DESCRIPTION
**Pull request by BiGsuw**
What is the purpose of the change
fix PythonTableUtils.getCollectionInputFormat cannot correctly handle None values
 , which is described in FLINK-27797

Does this pull request potentially affect one of the following parts:
Dependencies (does it add or upgrade a dependency): (no)
The public API, i.e., is any changed class annotated with @Public(Evolving): (no)
The serializers: (no)
The runtime per-record code paths (performance sensitive): (no)
Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (no)
The S3 file system connector: (no)
Documentation
Does this pull request introduce a new feature? (no)
If yes, how is the feature documented? (no)

### CODE REVIEW

**Comment by BiGsuw** [comment]
Hi @HuangXingBo 
   Can you help with the review?

File: flink-python/src/main/java/org/apache/flink/table/utils/python/PythonTableUtils.java [hunk 1]
```diff
@@ -512,7 +512,8 @@ private static Function<Object, Object> converter(
         }
 
         return c -> {
-            if (c.getClass() != byte[].class || dataType instanceof PickledByteArrayTypeInfo) {
+            if (c != null && c.getClass() != byte[].class
```
Comment (HuangXingBo) [inline flink-python/src/main/java/org/apache/flink/table/utils/python/PythonTableUtils.java:515]: If c== null, we should return c immediately.

**Review by HuangXingBo — COMMENTED** [review]
@BiGsuw Thanks a lot for the PR. I have left a comment.

File: flink-python/src/main/java/org/apache/flink/table/utils/python/PythonTableUtils.java (hunk 1 above)
Comment (BiGsuw) [reply flink-python/src/main/java/org/apache/flink/table/utils/python/PythonTableUtils.java:515]: @HuangXingBo  Thanks for your comment,
you're right,  this is a judgment that the data type is not what we want, I will change it now

**Pull request closed without merging**